index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css">
+  <title>Hello Microverse!</title>
+</head>
+<body>
+  <h2 class="h1">Hello Microverse!</h2>
+</body>
```

Comment:
- [ ] Let's follow the project's requirement by using the `h1` tag instead of `h2` tag. This is important because this element is the header of your page and `h1` tags tell search engine bots and web users what a page is about.